### ISSUE TITLE
Running on Google Colab

### ISSUE THREAD

**Issue by enzyme69** [ptrvilya/blendify#2]
In theory, we should be able to run this anywhere with Python 3.10 up right?

I am installing the module on Google Colab, then running the example cube,  but getting error:

```
/usr/local/lib/python3.10/dist-packages/numpy/core/getlimits.py:499: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/usr/local/lib/python3.10/dist-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
/usr/local/lib/python3.10/dist-packages/numpy/core/getlimits.py:499: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/usr/local/lib/python3.10/dist-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
---------------------------------------------------------------------------
UnsupportedOperation                      Traceback (most recent call last)
[<ipython-input-2-2ae244d3f684>](https://y4plpfm62xs-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230509-060147-RC00_530563781#) in <cell line: 2>()
      1 # Script to render cube
----> 2 from blendify import scene
      3 from blendify.materials import PrinsipledBSDFMaterial
      4 from blendify.colors import UniformColors
      5 # Add light

5 frames
[/usr/local/lib/python3.10/dist-packages/blendify/internal/io.py](https://y4plpfm62xs-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230509-060147-RC00_530563781#) in set_hook(self)
     21         self.logfile = open(self.logfile_path, 'w')
     22         self.logfile_fd = self.logfile.fileno()
---> 23         self.stdout_fd = sys.stdout.fileno()
     24         self.saved_stdout_fd = os.dup(self.stdout_fd)
     25         sys.stdout.flush()

UnsupportedOperation: fileno
```

**Comment by vguzov** [comment]
Hi, thanks a lot for reporting! We've actually never tested it on collab, but we'll look into it in the future version

**Comment by vguzov** [comment]
Fixed in version 1.2.3. I ran several examples on Collab, and I think everything should work correctly now. Let me know in case other problems arise!
